### PR TITLE
{cae,vis}[foss/2022b] VTK v9.2.6, meshio v5.3.4 w/ Python 3.10.8

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.7, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/3079c7385000e0e3deccc0a3acbafa68 for a full test report.

**Comment by Micket** [comment]
huh, i'm not sure what the bot tripped up on here 
```
== 2023-05-05 17:55:58,155 easyblock.py:3909 INFO Running method checksum_step part of step source
== 2023-05-05 17:55:58,351 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/easybuild-framework/easybuild/base/exceptions.py:126 in __init__): Invalid checksum spec 'None', should be a string (MD5) or 2-tuple (type, value). (at easybuild/easybuild-framework/easybuild/tools/filetools.py:1306 in verify_checksum)
== 2023-05-05 17:55:58,351 build_log.py:267 INFO ... (took < 1 sec)
```

File: easybuild/easyconfigs/v/VTK/VTK-9.2.6-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+source_urls = ['https://www.vtk.org/files/release/%(version_major_minor)s']
+sources = [
+    SOURCE_TAR_GZ,
+    '%(name)sData-%(version)s.tar.gz',
+]
+patches = [('vtk-version.egg-info', '.')]
+checksums = [
+    {'VTK-9.2.2.tar.gz': '1c5b0a2be71fac96ff4831af69e350f7a0ea3168981f790c000709dcf9121075'},
+    {'VTKData-9.2.2.tar.gz': '4b0ac438c77773ff741fd02c0e3b0dc1406b4c51e6cbe4e8304306fa125f27bf'},
```
Comment (Micket) [inline easybuild/easyconfigs/v/VTK/VTK-9.2.6-foss-2022b.eb:30]: oh, confusing error, but it's just that these checksums are wrong, and in particular, even the filenames are wrong.

**Review by Micket — DISMISSED** [review]
Please fix checksums

File: easybuild/easyconfigs/v/VTK/VTK-9.2.6-foss-2022b.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/v/VTK/VTK-9.2.6-foss-2022b.eb]: Not sure how that happened!  Hopefully all fixed now.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/e40faef27bebb132ad7b43871c11cd00 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**